packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -255,13 +278,21 @@ const Conversation = WebexPlugin.extend({
    * @param {Object} options
    * @returns {Promise<Conversation>}
    */
-  get(conversation, options) {
-    return this._inferConversationUrl(conversation)
-      .then(() => {
-        const {user, url} = conversation;
+  get(conversation, options = {}) {
+    const {user} = conversation;
+    let urlPromise;
 
-        options = options || {};
+    try {
+      const url = !user ? this.getConvoUrl(conversation) : '';
```

Comment:
Probably need to add to the jsdoc what all the different formats of conversation this method accepts.

Reply:
@adamweeks is it necessary to specify each type? Or is it okay as long as it conforms to this shape, i.e., `url`?

Reply:
@mattnorris  Just that it needs either an `id` or `url`. Also, it looks like if the conversation object has a `user` property, it sets the url to empty so it would be good to have that in the docs as a way this method supports it as well.

Reply:
I didn't add the `id` parameter because that is getting phased out. One shouldn't use a regular convo `id` to fetch a conversation